docs/attributes-registry/error.md

```diff
@@ -9,9 +9,12 @@ This document defines the shared attributes used to report an error.
 
 | Attribute | Type | Description | Examples | Stability |
 |---|---|---|---|---|
-| <a id="error-type" href="#error-type">`error.type`</a> | string | Describes a class of error the operation ended with. [1] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| <a id="error-message" href="#error-message">`error.message`</a> | string | A message providing more detail about an error in human-readable form. [1] | `TypeError: Cannot read properties of undefined (reading 'toString')` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
I'm supportive of this attribute but slightly worried about the duplication between `exception.*` and `error.*`. This PR does not change status quo, but makes the problem more obvious.

Could we come up with some litmus test on when to use `error.message` vs `exception.message` and add it to the note? Or at least say that semantic conventions  should wither avoid referencing both or should document the distinction and avoid duplicating the same info in both.

Also to this point, the example looks like `exception.message` - is there a reason we need a more generic `error.message` here?

Reply:
We need an error message for cases where an exception is not thrown. The case of feature flag evaluation failure for example. There may be e.g. a timeout but no exception thrown.

Reply:
makes sense, could we still add a note saying that it's not recommended to duplicate `error.message` and `exception.message`?

Reply:
sure

Reply:
I also changed the example to not appear to be a duplicate of an exception